reports/gp2gp_placeholder_trending_report/gp2gp_placeholders_breakdown_graph_trending_report_percentage.splunk

```diff
@@ -0,0 +1,30 @@
+index="$index$"
+
+{% from 'gp2gp_placeholder_trending_report/gp2gp_placeholder_trending_report_base.splunk' import
+gp2gp_placeholder_report_trending_base %}
+
+{{ gp2gp_placeholder_report_trending_base() }}
+
+| WHERE numPlaceholders > 0
+| spath output=placeholders path=payload.ehr.placeholders{}
+
+
+| eval numPlaceholdersPerTransfer = case(
+    numPlaceholders >=1 AND numPlaceholders <=5, "1-5 placeholders",
+    numPlaceholders >=6 AND numPlaceholders <=10, "6-10 placeholders",
+    numPlaceholders >=11 AND numPlaceholders <=15, "11-15 placeholders",
+    numPlaceholders >=16 AND numPlaceholders <=20, "16-20 placeholders",
+    numPlaceholders >=21, "21+ placeholders")
+
+| eval time_period = case(
+                          "$time_period$"=="month", strftime(_time,"%Y-%m"),
+                          "$time_period$"=="week", strftime(_time,"%Y-Wk%W"),
+                          "$time_period$"=="day", strftime(_time,"%Y-%m-%d")
+                          )
+
+| chart count by time_period, numPlaceholdersPerTransfer
+
+| addtotals fieldname="total_per_month"
+| foreach * [eval <<FIELD>>=if("<<FIELD>>" == "time_period", time_period, round('<<FIELD>>'/total_per_month*100, 2))]
+| fields time_period, "1-5 placeholders", "6-10 placeholders", "11-15 placeholders", "16-20 placeholders", "21+ placeholders"
+| fields - total_per_month
```

Comment:
does this need to go before line 29? or does line 29 get rid of this field and thus this line is redundant?

Reply:
I don't think the order matters.
Line 29 specifies the order of the columns, but `total_per_month` was still being displayed which is why I added line 30.